util/cache.c

```diff
@@ -11,27 +12,23 @@
 #define LINE_START(a) ROUND_DOWN(a, CONFIG_L1_CACHE_LINE_SIZE_BITS)
 #define LINE_INDEX(a) (LINE_START(a)>>CONFIG_L1_CACHE_LINE_SIZE_BITS)
 
-static inline void
-dsb(void)
+static inline void dsb(void)
 {
     asm volatile("dsb sy" ::: "memory");
 }
 
-static inline void 
-dmb(void)
+static inline void dmb(void)
```

Comment:
I know it's a style fight issue, but I much prefer the function name on a line on its own.  Not a show stopper for this PR.